tests/lldp/test_lldp.py

```diff
@@ -39,7 +39,8 @@ def restart_swss_container(duthosts, enum_rand_one_per_hwsku_frontend_hostname,
 
     # make sure all critical services are up
     assert wait_until(600, 5, 30, duthost.critical_services_fully_started), (
-        "Not all critical services are fully started after restarting orchagent. "
+        "Not all critical services started after restarting orchagent. "
+        "Service status: {}".format(duthost.get_critical_services_status())
     )
```

Comment:
There is no function `get_critical_services_status`, why do you modify code like this? There is description in this assertion, why did you want to add more for this summary?